### PR TITLE
Generate template

### PR DESCRIPTION
**Pull request by neonichu**
- The code for the conversion is pretty much done, it requires contentful/contentful.rb#76 and contentful/contentful.rb#75, though, so `contentful.rb` master at this point. 

- The spec requires network at the moment :cry: 

- There's no way to access the functionality from the CLI. Currently, the assumption that one parameter is required is hardcoded, the generator needs two, though (space id and access token)

TODO:

- [x] Wait for release of contentful.rb with required fixes
- [x] Make spec not use the network
- [x] Integrate into CLI

@dlitvakb maybe you want to finish this? ;)

### CODE REVIEW

**Comment by dlitvakb** [comment]
@neonichu almost done, we need to pull in @pxlpnk and make a contentful.rb release

**Comment by dlitvakb** [comment]
@neonichu would you like to give this a quick review?

I finished everything required for this to work :dancers:

File: lib/contentful/bootstrap/commands.rb [hunk 1]
```diff
@@ -105,6 +106,29 @@ def generate_token(space, options = {})
...
+
+        if access_token.nil?
+          puts "Access Token not specified"
+          puts "Exiting!"
+          exit
+        end
+
+        puts "Generating JSON Template '#{filename}' for Space: '#{space_id}'"
+        puts "If you've indicated a filename, JSON will be automatically saved to that file"
+        puts
```
Comment (neonichu) [inline lib/contentful/bootstrap/commands.rb]: I think it would be preferable to have different messages when writing to a file or to stdout.

**Comment by neonichu** [comment]
We should add CLI help for the `generate_json` command, at the moment it is wrong or non-existing, depending on what the user does:

```bash
$ contentful_bootstrap generate_json 
Usage: contentful_bootstrap <command> <space> [options]

You must provide a valid space name. Exiting!
$ contentful_bootstrap generate_json —help
```

Could we also switch on Travis, so that the specs are actually continuously run?

**Comment by dlitvakb** [comment]
@neonichu all fixed :+1:

**Pull request merged**